lib/litle/api.rb

```diff
@@ -70,11 +70,22 @@ def credit_payment(kb_account_id, kb_payment_id, kb_payment_transaction_id, kb_p
       end
 
       def refund_payment(kb_account_id, kb_payment_id, kb_payment_transaction_id, kb_payment_method_id, amount, currency, properties, context)
-        # Pass extra parameters for the gateway here
-        options = {}
+        gateway_call_proc = Proc.new do |gateway, linked_transaction, payment_source, amount_in_cents, options|
+          gateway.refund(amount_in_cents, linked_transaction.txn_id, options)
+        end
 
-        properties = merge_properties(properties, options)
-        super(kb_account_id, kb_payment_id, kb_payment_transaction_id, kb_payment_method_id, amount, currency, properties, context)
+        linked_transaction_proc = Proc.new do |amount_in_cents, options|
+          # we need the id of either a capture or sale transaction
```

Comment:
Interesting - do you know if that has changed? I'm surprised it hasn't been caught earlier.

Is there a specific certification test we could add for regression?